src/index.js

```diff
@@ -1,9 +1,8 @@
-// import _ from 'lodash';
+import AddItem from './modules/addItem.js';
 import './style.css';
-import todotasks from './todotasks.js';
-import Refresh from './refresh.svg';
-import Enter from './enter.svg';
-import Menu from './menu.svg';
+import Refresh from './assets/refresh.svg';
+import Enter from './assets/enter.svg';
+// import Tasks from './modules/tasks.js';
```

Comment:
Great job @PRATAP-KUMAR 🤝

- [ ] [Optional] Kindly remove all commented code from your project. This is not regarded as a good practice.